webapp/src/components/modals/channel_settings/index.ts

```diff
@@ -24,9 +25,11 @@ import ChannelSettingsModal from './channel_settings';
 const mapStateToProps = (state) => {
     const channelId = getChannelIdWithSettingsOpen(state);
     let channel = null;
+    let isChannelDmOrGm = false;
```

Comment:
naming: 1/5 `s/isChannelDmOrGm/omitDisplayName/` - more intention revealing

Reply:
That's a much better choice.  Commit the suggestion.  Thanks, @levb